### ISSUE TITLE
what's the meaning of the average score in rank:pairwise objective?

### ISSUE THREAD

**Issue by chloe-wang** [microsoft/LightGBM#2894]
I observed that the average value  of the predicted scores is around 0.5. Because rank:pairwise only considers the relavant value between two samples in loss function, how can we get the average value of the predicted scores?
